common/acknowledgements/aria-wg-active.html

```diff
@@ -1,99 +1,76 @@
 <section class="section" id="ack_group">
 	<h4>Participants active in the ARIA WG at the time of publication</h4>
 	<ul>
-		<li>Irfan Ali (Educational Testing Service)</li>
-		<li>CB Averitt (Deque Systems, Inc)</li>
 		<li>Sina Bahram (Invited Expert)</li>
-		<li>Shirisha Balusani (Invited Expert)</li>
```

Comment:
Just to double check: was Shirisha Balusani removed entirely or was it a name change to Shirisha Gubba?

Reply:
@shirsha Please confirm the name change is correct.